doc/historylog.rst

```diff
@@ -2,26 +2,45 @@
 History
 #######
 
-With the **History** pane, you can view an automatically de-duplicated, time-stamped list of every command you enter into any connected :doc:`ipythonconsole`.
+With the **History** pane, you can view all the commands you've entered into any :doc:`ipythonconsole`, along with their timestamp.
 
-.. image:: images/history/history-menu.png
+.. image:: images/history/history-standard.png
    :alt: Spyder History Log, displaying a list of previously executed commands
 
 
 
-=====================
-Using the History Log
-=====================
+======================
+Using the History pane
+======================
 
-Navigating the :guilabel:`History Log` is very straightforward.
-Each session is marked by a date and time-stamp, making it easy to remember when you executed a certain command.
+Navigating the :guilabel:`History` pane is very straightforward.
+Each Spyder session is marked by a date and timestamp, making it easy to remember when you executed a certain command.
 Statements can be selected and copied from the context menu or with the normal system shortcuts.
-Just like in the editor, highlighting a word or phrase displays all other occurrences, and full syntax highlighting is also supported.
-Finally, the top-right options menu (:guilabel:`Gear` icon) allows you to toggle soft-wrapping of long lines (:guilabel:`Wrap lines`), and set the number of commands the :guilabel:`History Log` should remember (:guilabel:`History`).
+Just like in the Editor, selecting a word or phrase displays all other occurrences, and full syntax highlighting is also supported.
 
-The :guilabel:`History Log` is stored in the :file:`.spyder-py3` (Python 3) or :file:`spyder` (Python 2) directory in your user home folder (by default, :file:`C:/Users/{username}` on Windows, :file:`/Users/{username}` for macOS, and typically :file:`/home/{username}` on GNU/Linux).
+
+
+============
+Options Menu
+============
+
+The top-right options menu (:guilabel:`Hamburger` icon) allows you to toggle wrapping of long lines (:guilabel:`Wrap lines`), and whether the line number is displayed to the left of the text (:guilabel:`Show line numbers`).
+
+.. image:: images/history/history-wrap.gif
+   :alt: Spyder History Log, displaying wraping lines and showing line number
+
+
+==============
+Advanced usage
+==============
+
+The list of commands shown in the :guilabel:`History` pane are stored in :file:`history.py` in the :file:`.spyder-py3` directory in your user home folder (by default, :file:`C:/Users/{username}` on Windows, :file:`/Users/{username}` for macOS, and typically :file:`/home/{username}` on GNU/Linux).
 You might need to show invisible files in order to see it on a non-Windows operating system.
 
+.. image:: images/history/history-log-file.png
```

Comment:
Could you use the light macOS theme? The dark theme is non-standard and a bit hard to read.

Reply:
- [ ] Its an OS, not Spyder level setting on a minority OS, and likely a majority of developers prefer the dark theme, so I don't see the "non-standard" nature of the dark theme (built-in to the OS and heavily promoted by Apple) as an issue. However, I do agree the text for invisible items is quite hard to read with it enabled, so I guess its worth changing for this particular screenshot.
- [ ] It would be much better to show the file in a real editor with syntax highlighting, so its clearer its just a normal Python file rather than some bespoke text format.
- [ ] The size of the screenshot, at nearly 2 MB, is gargantuan for what it shows, larger than most GIFs. Downscaling by 2x and applying `optipng` results in a size of only 0.35 MB, less than 1/5 of the original, for essentially no loss in perceptible quality at the final displayed resolution. Again, to make things simplest, I can do this as part of the rebase pass once you've updated the screenshot accordingly.

Reply:
> Its an OS, not Spyder level setting on a minority OS, and likely a majority of developers prefer the dark theme, so I don't see the "non-standard" nature of the dark theme

I disagree. It took me a long time (i.e. several seconds) to understand what that screenshot was about. I didn't get that a Finder instance was displayed at the left until I paid close attention to the window the details.

Also, please remember that our docs are for people that are just getting started with Spyder. Of course it can be useful for experienced developers too, but that's not our main target.

Finally, macOS can have a small market share but most tutorials and videos are done with it. So I think most people are accustomed to see Mac screenshots here and there.

> However, I do agree the text for invisible items is quite hard to read with it enabled, so I guess its worth changing for this particular screenshot.

Yep, that's another important issue.

> It would be much better to show the file in a real editor with syntax highlighting

Agreed. You can use Sublime for that @juanis2112.

> The size of the screenshot, at nearly 2 MB, is gargantuan for what it shows

Good point too.

Reply:
> I disagree. It took me a long time (i.e. several seconds) to understand what that screenshot was about. I didn't get that a Finder instance was displayed at the left until I paid close attention to the window the details.

Okay, then if it is an issue with familiarity, that makes sense then. Thanks for explaining.

Reply:
Its using a real editor now, but the session timestamp (`############ Date ##########` that the history log adds) is not visible anymore, so its not clear its a history log file. I would suggest showing that. Also, it looks much clearer to me what's going on, though as far as I can tell, it looks like its still using the dark theme so I'm not sure if @ccordoba12 still thinks it should be changed (but I would assume yes).

Reply:
> I would suggest showing that

Agreed.

> I'm not sure if @ccordoba12 still thinks it should be changed (but I would assume yes).

Correct, I still prefer to see Finder on the left with the Mac light theme.

Reply:
@juanis2112 Should be last two things on this PR, other than the line break and image stuff I'll fix:

* [ ] Scroll editor to the top of the file to show the session timestamp, etc.
* [ ] Use light OS theme

Reply:
LGTM now.